lattices/LatticeMath/LatticeStatsDataProvider.tcc

```diff
@@ -147,6 +157,12 @@ void LatticeStatsDataProvider<T>::setLattice(
 		_currentSlice.assign(lattice.get());
 		_atEnd = False;
 	}
+#ifdef _OPENMP
+	_nMaxThreads = min(
+	    omp_get_max_threads(),
+	    lattice.size()/ClassicalStatisticsData::BLOCK_SIZE + 1
```

Comment:
A minor point, but assuming that the value should be ceil(lattice.size()/BLOCK_SIZE), wouldn't this value be marginally better if it were (lattice.size() + BLOCK_SIZE - 1) / BLOCK_SIZE?